Misc/NEWS.d/next/Security/2024-02-18-03-14-40.gh-issue-115398.tzvxH8.rst

```diff
@@ -0,0 +1,7 @@
+Allow controlling Expat >=2.6.0 reparse deferral (CVE-2023-52425) by adding
+four new methods:
+
+* ``pyexpat.xmlparser.SetReparseDeferralEnabled``
+* ``xml.etree.ElementTree.XMLParser.flush``
+* ``xml.etree.ElementTree.XMLPullParser.flush``
```

Comment:
The should be documented in the module documentations.

Please add also a What's New entry.

Reply:
> The should be documented in the module documentations.

@serhiy-storchaka understood, I would like to keep that for last after the rest iof the pull request is already at an agreeable point.

> Please add also a What's New entry.

Do you have any links on the where and how of that? I haven't done that before. It doesn't seem to involve blurp?

Reply:
It is the `Doc/whatsnew/3.13.rst` file.

The documentation for new methods should be added in `Doc/library/pyexpat.rst` and `Doc/library/xml.etree.elementtree.rst` files. Search where other methods of corresponded classes are documented.

Reply:
@serhiy-storchaka I see, thanks!